### PR TITLE
B/search description

### PR DESCRIPTION
**Pull request by tomwayson**
1. Description:

use searchDescription for content.summary, not content.description

also fixes tests for relativeDateToDateRange() that would only fail at the end of months that are longer than the previous month. Not sure if we consider the underlying logic a bug, but wasn't sure what logic we want. For example: what should be the from date of the range that represents the previous 1 month form 3/30? There is no 2/30. So I kept the existing logic under the assumption that if it's good enough for `Date.setDate()` it's good enough for us, and just updated the test expectations.

1. Instructions for testing:

1. Closes Issues: [3507](https://devtopia.esri.com/dc/hub/issues/3507)

1. [x] ran commit script (`npm run c`)

### CODE REVIEW

File: packages/common/src/search/utils.ts [hunk 1]
```diff
@@ -218,6 +218,9 @@ export function relativeDateToDateRange(
       break;
     case "months":
       // get the current month and subtract num
+      // NOTE: when the previous month has fewer days than this month
+      // setMonth() will return a date w/in the current month
+      // example: 3/30 -> 3/2 b/c there is no 2/28
```
Comment (tomwayson) [inline packages/common/src/search/utils.ts:223]: bah, there is no 2/30

Comment (dbouwman) [reply packages/common/src/search/utils.ts:223]: dates-- thanks for adding this note and fixing the test

**Review by dbouwman — APPROVED** [review]
Code looks good to me, but I'm not exactly sure about the expected logic. I assume @sonofflynn89  is covering that

**Review by sonofflynn89 — APPROVED** [review]
Good work!

**Pull request merged**